packages/react-cards/src/components/Card/CardItem/CardItem.types.ts

```diff
@@ -1,6 +1,7 @@
 import { IComponentStyles, IHTMLSlot, IComponent, IStyleableComponentProps } from '@uifabric/foundation';
+import { IStackItemProps } from 'office-ui-fabric-react';
 
-export type ICardItemComponent = IComponent<ICardItemProps, ICardItemTokens, ICardItemStyles>;
+export type ICardItemComponent = IComponent<ICardItemProps & IStackItemProps, ICardItemTokens, ICardItemStyles>;
```

Comment:
It might be safer to have `ICardItemProps` extend `IStackItemProps`, but what I think might even be better is to try and get rid of `ICardItemProps` altogether (per my other comment.) Are any of the types in these files needed or can they all be `Stack` types/

Reply:
I was going to comment asking why not do:

>It might be safer to have ICardItemProps extend IStackItemProps

I'm interested to see what the resolution is here, for my own knowledge at the very least!

Reply:
Mak and I talked more about this offline so I'll just reiterate a bit here. I think at the very least it's a good idea to remove the union. There are some shared props that have conflicting types and this could get nastier over time. I think there's a chance a `CardItem` is basically a specialized `StackItem` that may not need its own prop types.

Reply:
I just pushed the changes to get rid of all of the `Card` types altogether.